webextensions/_locales/fr/messages.json

```diff
@@ -133,250 +133,254 @@
-  "message_startup_description_2": { "message": " ou cliquer sur " },
-  "message_startup_description_3": { "message": " button in the toolbar, to activate the sidebar panel." },
+  "message_startup_description_1": { "message": "La barre d'onglets de Tree Style Tab est l'un des choix de panneau latéral disponibles. Si vous ne le voyez pas encore, appuyez sur la " },
+  "message_startup_description_key": { "message": "touche « F1 »" },
+  "message_startup_description_2": { "message": " ou cliquez sur le bouton " },
+  "message_startup_description_3": { "message": " dans la barre d'outils, pour activer le panneau latéral." },
 
-  "message_startup_requestPermissions_description": { "message": "⚠Il y a quelques fonctionnalités qui requierent des permissions supplémentaires. Si vous voulez les utiliser, acceptez les permissions manuellement." },
-  "message_startup_requestPermissions_bookmarks": { "message": "\"Marquer cette arborescence\" dans le menu contextuel sur les onglets (requiert l'accès aux marque-page)" },
-  "message_startup_requestPermissions_allUrls":   { "message": "Don't expand collapsed tree and skip collapsed descendants, while switching focus of tabs via keyboard shortcuts (requires script injection to webpages for detection of keyboard operations)" },
+  "message_startup_requestPermissions_description": { "message": "⚠Quelques fonctionnalités requièrent des permissions supplémentaires. Si vous voulez les utiliser, acceptez les permissions manuellement." },
+  "message_startup_requestPermissions_bookmarks": { "message": "« Ajouter cet arbre aux marque-pages » dans le menu contextuel sur les onglets (requiert l'accès aux marque-pages)" },
+  "message_startup_requestPermissions_allUrls":   { "message": "Ne pas déplier les arbres repliés et ignorer les descendants repliés lorsque vous changez d'onglet courant via des raccourcis claviers (nécessite une injection de script dans les pages web pour détecter les appuis de touche)" },
 
-  "message_startup_userChromeCss_notify": { "message": "How to hide top tab bar and sidebar header?" },
-  "message_startup_userChromeCss_description_1": { "message": "Due to restrictions of WebExtensions, Tree Style Tab cannot control Firefox's top tab bar and the sidebar header. If you need to control them, see " },
-  "message_startup_userChromeCss_description_link_label": { "message": "exemples de customisation via userChrome.css" },
+  "message_startup_userChromeCss_notify": { "message": "Comment cacher la barre d'onglets du haut et l'entête du panneau latéral ?" },
+  "message_startup_userChromeCss_description_1": { "message": "À cause de restrictions de WebExtensions, Tree Style Tab ne peut pas modifier la barre d'onglets du haut ou l'entête du panneau latéral de Firefox. Si vous voulez les modifier, consultez les " },
+  "message_startup_userChromeCss_description_link_label": { "message": "exemples de personnalisation via userChrome.css" },
```

Comment:
Missing caps?

Reply:
No, it’s several strings that are concatenated together with sometimes markup on top of it.